extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelConfig.java

```diff
@@ -62,6 +62,13 @@
         @ConfigItem(defaultValue = "true")
         public boolean enabled;
 
+        /**
+         * If {@code true}, the camel context will be started at initialization time
+         * to boot up faster.
+         */
+        @ConfigItem
+        public boolean lightweight;
```

Comment:
Wonder if `static` would be better as it would better reflect when it si initialized as well as the fact that it cannot be changed once configured ?

Reply:
{{static}} is a reserved word and can't be used as a property name...

Reply:
Ah yeah, let maybe name it `static_` (as we do for the native one) and name the config item `static` ?

in any case, I don't really care much :)